### ISSUE TITLE
Show better status when Reconciling/Transitioning of backend resource underway

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#4674]
When we're reconciling or transitioning a cluster on Kubernetes, the UI removes a link to that cluster until it becomes available again. This makes the use of particular pages unclear since the user can't access what they think is available, when in fact it isn't.

To help better allow users to know that a cluster isn't up quite yet, we should provide a better status -- tooltip or banner -- indicating that the link disappeared with an appropriate reason about what's happening on the backend. I've been told that in our code, we do have an `isReady` condition that we could use, which may be already doing something in hiding the link.

**Comment by catherineluse** [comment]
@Tejeev do you think this is related to the issue you saw where the cluster link was missing?

**Comment by Tejeev** [comment]
yes, definitely related.  However, Iirc the customer i was working with just can't reach their cluster in vue though they can in the old UI
